platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -44,24 +35,16 @@ public void configurationEvent(ConfigurationEvent event) {
             () -> {
               try {
                 String type = getType(event);
-                Configuration[] configuration =
-                    configurationAdmin.listConfigurations("(service.pid=" + event.getPid() + ')');
-
-                String updatedConfiguration = "{}";
-                // the service.pid is unique so we just need to check if we got anything.
-                if (configuration != null && configuration.length > 0) {
-                  updatedConfiguration = dictionaryToString(configuration[0].getProperties());
-                }
 
                 // check if there is a subject associated with the configuration change
                 if (ThreadContext.getSubject() != null
                     || Subject.getSubject(AccessController.getContext()) != null) {
-                  SecurityLogger.audit("Configuration {}: {}", type, updatedConfiguration);
+                  SecurityLogger.audit("Configuration {} for {}.", type, event.getPid());
                 } else {
                   // there was no subject change was caused by an update to the config file on the
                   // filesystem
                   SecurityLogger.auditWarn(
-                      "Configuration {} via filesystem: {}", type, updatedConfiguration);
+                      "Configuration {} via filesystem for {}.", type, event.getPid());
                 }
               } catch (Throwable e) {
```

Comment:
✏️ I don't believe you need this catch clause anymore. Which should also mean you don't need the suppression anymore.